sdk/formrecognizer/ai-form-recognizer/test/public/node/analysis.spec.ts

```diff
@@ -0,0 +1,746 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { env, Recorder } from "@azure-tools/test-recorder";
+import { matrix } from "@azure/test-utils";
+import { assert } from "chai";
+import fs from "fs";
+import { Context } from "mocha";
+import path from "path";
+import {
+  DocumentAnalysisClient,
+  DocumentModelAdministrationClient,
+  IdentityDocument,
+  ModelInfo,
+  PrebuiltModels,
+} from "../../../src";
+import { DocumentDateField, DocumentSelectionMarkField } from "../../../src/models/fields";
+import { createRecorder, makeCredential, testPollingOptions } from "../../utils/recordedClients";
+
+const endpoint = (): string => env.FORM_RECOGNIZER_ENDPOINT;
+
+function makeTestUrl(urlPath: string): string {
+  const testingContainerUrl = env.FORM_RECOGNIZER_TESTING_CONTAINER_SAS_URL;
+  const parts = testingContainerUrl.split("?");
+  return `${parts[0]}${urlPath}?${parts[1]}`;
+}
+
+function assertDefined(value: unknown, message?: string): asserts value {
+  return assert.ok(value, message);
+}
+
+matrix([[false]] as const, async (useAad) => {
```

Comment:
why not true too?

Reply:
AAD hadn`t worked for a while. I`m testing this locally.

Reply:
Tested, recorded, and updated this to `true, false`.